mmdet/models/detectors/maskformer.py

```diff
@@ -43,6 +43,9 @@ def __init__(self,
         self.train_cfg = train_cfg
         self.test_cfg = test_cfg
 
+        if self.num_stuff_classes > 0:
+            self.show_result = self._show_pan_result
```

Comment:
Need to  add comments to explain that we use `SingleStageDetector.show_result` for instance segmentation.

Reply:
Comment added.

Reply:
Sorry, I misremembered, it's `BaseDetector.show_result` instead of `SingleStageDetector.show_result`.

Reply:
Good catch, updated.